src/store/modules/auth/epic/signOutEpic.js

```diff
@@ -0,0 +1,20 @@
+import { Observable } from 'rxjs'
+import { storage } from '../../../../utils'
+import { STORAGE_RECORD_KEY } from '../constants'
+import { SIGN_OUT } from '../actionsTypes'
+
+function handleSignOut () {
+  storage.remove(STORAGE_RECORD_KEY)
+}
+
+function mapToSignOutAction$ () {
+  return Observable.empty()
+}
+
+const signOutEpic = action$ =>
```

Comment:
to be consistent, consider removing token from storage in `${SIGN_IN}/${STATUS_SUCCESS}` epic. take a look at sign in epics code for reference

Reply:
disregard this